CODEOWNERS

```diff
@@ -12,6 +12,8 @@
 *       @GITHUB_USER_NAME_GOES_HERE
```

Comment:
This is invalid FWIW.

Reply:
good catch. fixed: https://github.com/TBD54566975/web5-go/commit/06cb0b190d19dedf77d4ee60216745768cc9f2a8